apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx

```diff
@@ -133,13 +133,13 @@ const GetExternalProviders = (): React.ReactElement =>
     name: 'getExternalProviders',
     title: 'Get External Providers',
     label: 'excludeAddedProviders',
-    onClick: async (excludeAddedProviders: boolean) => {
+    onClick: async (excludeAddedProviders: boolean, setResult: (result: string) => void) => {
       let result;
```

Comment:
Is this really used anymore?

Reply:
Yes, For the promise based apis which still uses callback

Reply:
I get that for the generic case, but in this particular one, you're doing `let result` and at the end `return result` without actually assigning anything to it.